crypto/sigpb/sigpb.proto

```diff
@@ -44,10 +44,24 @@ message DigitallySigned {
     ECDSA = 3;
   }
 
+  // SignatureCipherSuite defines the set of algorithms used for signing.
+  enum SignatureCipherSuite {
```

Comment:
It used to be at the top level, but I thought it should be specific to signature generation so I moved it here.  Perhaps other contexts should define their own enums?